### DISCUSSION TITLE
Awesome project

### DISCUSSION THREAD

**Discussion by saashqdev** [wansenai/wansen-erp#196] [General]
Just thought I'd drop you a note. This is a great looking project. If you've got any boring, repetitive stuff I can give you a hand.
Cheers, Dave

**Comment by Jzow** [comment]
Hi Dave, I'm glad you found this project. It's currently in its early stages and requires a lot of work in the front-end. 

You can take a look at #118 and choose the one you're interested in to try. If you have any questions, we can communicate with you. Thank you!
